packages/core/src/components/TextField/Mask.jsx

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * A `Mask` component renders a controlled input field. When the
+ * field is blurred, it applies formatting to improve the readability
+ * of the value. When the field is focused, formatting is removed
```

Comment:
I don't see formatting being removed on focus (I assume that the commas are meant to be removed, right?)

Reply:
Ah good catch. This is actually the expected behavior and this comment is incorrect.